### ISSUE TITLE
Cannot use java.home from settings.json if spaces are included

### ISSUE THREAD

**Issue by agiles231** [adashen/vscode-tomcat#311]
- VSCode Version: 1.48.1
3dd905126b34dcd4de81fa624eb3a8cbe7485f13
x64
- OS Version: Windows 10 Enterprise
- Tomcat Extension Version: 0.11.3

Steps to Reproduce:

1. Assign java.home a value with a space (e.g. "C:\\Program Files\\Java\\jdk-14")
2. Try to run Tomcat through extension

Command will fail saying "'C:\Program' is not recognized as an internal or external command, operable program or batch file."

However, if JAVA_HOME environment variable contains the same value, tomcat will start fine through extension.

**Comment by spdermn02** [comment]
not that it doesn't need to be fixed but try
"C:\\PROGRA~1\\Java\\jdk-14"
in the java.home value.

**Comment by Yeshuaihui** [comment]
> not that it doesn't need to be fixed but try
> "C:\PROGRA~1\Java\jdk-14"
> in the java.home value.

是的 我这么做了，但是现在是" The system cannot find the path specified"的异常了

**Comment by Yeshuaihui** [comment]
> not that it doesn't need to be fixed but try
> "C:\PROGRA~1\Java\jdk-14"
> in the java.home value.

Yes, I did, but now it's the exception of "The system cannot find the path specified"
